editor/editor_help.cpp

```diff
@@ -2097,18 +2103,26 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 			tag_stack.pop_front();
 			pos = brk_end + 1;
 			if (tag != "/img") {
-				p_rt->pop();
+				p_rt->pop(); // font
 				if (code_tag) {
 					p_rt->pop(); // font size
-					// Pop both color and background color.
-					p_rt->pop();
-					p_rt->pop();
+					p_rt->pop(); // color
+					p_rt->pop(); // background color
 				} else if (codeblock_tag) {
 					p_rt->pop(); // font size
-					// Pop color, cell and table.
-					p_rt->pop();
-					p_rt->pop();
-					p_rt->pop();
+
+					// Copy codeblock button.
+					p_rt->push_cell();
+					p_rt->set_cell_row_background_color(code_bg_color, code_bg_color);
```

Comment:
This doesn't use exactly the same color for the second argument. For some reason we set up the first cell with the even color being at 0.99 alpha. Not sure why it's relevant, but we should change either that line or your addition, so they are the same.